test/end-to-end/pages/applicant/address.js

```diff
@@ -1,17 +1,31 @@
 'use strict';
 
-const commonContent = require('app/resources/en/translation/common');
+const commonContentEn = require('app/resources/en/translation/common');
+const commonContentCy = require('app/resources/cy/translation/common');
 const pageUnderTest = require('app/steps/ui/applicant/address/index');
 
-module.exports = function () {
+async function enterApplicantAddressManually(language = 'en', testAddressIndex) {
+
+    const commonContent = language === 'en' ? commonContentEn : commonContentCy;
+    const findAddress = language === 'en' ? 'Find UK address' : 'Dod o hyd i gyfeiriad';
     const I = this;
 
-    I.seeCurrentUrlEquals(pageUnderTest.getUrl());
-    I.click('.summary');
-    I.fillField('#addressLine1', 'test address for applicant line 1');
-    I.fillField('#addressLine2', 'test address for applicant line 2');
-    I.fillField('#addressLine3', 'test address for applicant line 3');
-    I.fillField('#postTown', 'test address for applicant town');
-    I.fillField('#newPostCode', 'postcode');
-    I.waitForNavigationToComplete(`input[value="${commonContent.saveAndContinue}"]`);
-};
+    if (!testAddressIndex) {
+        testAddressIndex = '0';
+    }
+
+    I.waitInUrl(pageUnderTest.getUrl());
+    I.seeInCurrentUrl(pageUnderTest.getUrl());
+    I.fillField('postcode', 'SW9 9PD');
+    await I.navByClick(findAddress);
+    I.wait(2);
+    I.waitForVisible('#postcodeAddress');
+    I.selectOption('#postcodeAddress', testAddressIndex);
+    I.wait(2);
+    I.waitForElement('#addressLine1');
```

Comment:
don't need I.wait(2), use e.g. await I.waitForElement(#addressLine1, config.TestTimeToWaitForElement), with a max time in TestTimeToWaitForElement